.github/ISSUE_TEMPLATE/research.md

```diff
@@ -0,0 +1,22 @@
+---
+name: 🍎 Research
```

Comment:
Apple, in English, commonly means an apple. I would _strongly_ advise against using this emoji, and instead use the research emoji.

Reply:
Unfortunately no such emoji exists, though some alternatives could be:

:microscope:
:scientist: 
:nerd_face: 

And potentially others.

Reply:
Let's make a ticket for this...

Reply:
A research ticket?